src/main/kotlin/dev/rosewood/roseskyblock/command/argument/IslandWorldGroupArgument.kt

```diff
@@ -20,6 +20,7 @@ import java.util.function.BiFunction
  *
  * @param <C> Command sender type
  */
+// not sure about this being a builder setup but I'll leave it for now
```

Comment:
Remove comment

Reply:
The comment was put there for you to read and then remove silly goose